src/main/kotlin/fi/metatavu/pakkasmarja/services/erp/sap/AbstractSapResourceController.kt

```diff
@@ -28,9 +28,14 @@ abstract class AbstractSapResourceController {
      * @param routeId SAP session route id
      * @return created item
      */
-    fun createItem(item: JsonNode, resourceUrl: String, sessionId: String, routeId: String): JsonNode {
+    fun createItem(
```

Comment:
JsonNode is used across the application to describe SAP resources, they all should be replaced by Jackson models instead. 

I'll pause the review until this has been fixed and resume when the change is complete